storage/2020-08-04/blob/blobs/immutability_policy_blob.go

```diff
@@ -0,0 +1,128 @@
+package blobs
+
+import (
+	"context"
+	"net/http"
+	"strings"
+
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/azure"
+	"github.com/Azure/go-autorest/autorest/validation"
+	"github.com/tombuildsstuff/giovanni/storage/internal/endpoints"
+)
+
+type ImmutabilityPolicyBlobInput struct {
+	UntilDate  *string
+	PolicyMode *string
+}
+
+func (client Client) SetImmutabilityPolicyBlob(ctx context.Context, accountName, containerName, blobName string, input ImmutabilityPolicyBlobInput) (result autorest.Response, err error) {
```

Comment:
could we update the `lifecycle_test.go` to include setting/retrieving/deleting a policy?